bugbug/repository.py

```diff
@@ -233,8 +234,9 @@ def _hg_log(revs):
 
     revs = []
     for rev in hglib.util.grouper(template.count("\\0"), out):
-        posixtime = float(rev[3].split(b".", 1)[0])
-        dt = datetime.fromtimestamp(posixtime)
+        date = datetime.fromtimestamp(float(rev[3].split(b".", 1)[0]))
```

Comment:
You should probably use utcfromtimestamp too here else you'll have a local time here.

Reply:
I copied this from hglib, so I assumed that's the local timestamp

Reply:
Fixed with 3fd81a928eebe8c622784c9742e942a0444a8c35.